packages/axelar-wasm-std/src/address.rs

```diff
@@ -40,6 +41,16 @@ pub fn validate_address(address: &str, format: &AddressFormat) -> Result<(), Err
             ScAddress::from_str(address)
                 .change_context(Error::InvalidAddress(address.to_string()))?;
         }
+        AddressFormat::Base58Solana => {
+            const SOLANA_PUBKEY_LEN: usize = 32;
+
+            let pubkey_vec = bs58::decode(address)
+                .into_vec()
+                .change_context(Error::InvalidAddress(address.to_string()))?;
+            if pubkey_vec.len() != SOLANA_PUBKEY_LEN {
+                bail!(Error::InvalidAddress(address.to_string()))
+            }
+        }
```

Comment:
where is this being used?

Reply:
It shouldn't be part of this branch (unrelated changes), we removed it at https://github.com/axelarnetwork/axelar-amplifier/pull/744/commits/00ed5e387220920def0e22a31e9793f4f932c213